### PR TITLE
Search design review

### PR DESCRIPTION
**Pull request by davidrapson**
NP-1025

- Fixes search input to standard 44px interactive target size
- Normalises button styles by only sharing common styles with main buttons rather than attempting to override and unset lots of properties

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 **Bugfixes**:
 
+* Search: Normalise search button styles and use 44px height. Removed `cads-button` from search input.
```
Comment (baseonmars) [inline CHANGELOG.md:12]: This needs to go at the top of the files, before the version for the previous release.

**Review by baseonmars — APPROVED** [review]
Lovely and clean! Changelog needs to be fixed up, then go ahead and merge.

**Comment by davidrapson** [comment]
@soniaturcotte Mind checking out this branch and giving these changes a quick review?

**Review by luke-hill — APPROVED** [review]
Might need to tweak stuff on public-website-cucumber when this goes live. Not 100% sure. But DS is all fine.

**Pull request merged**